mm_cli.py

```diff
@@ -0,0 +1,63 @@
+#! /usr/bin/env python3
+# CLI interface for mgamcmahon program
+
+import argparse
+import mcmahon
+import yaml
```

Comment:
See PEP8 on organizing imports. Should be builtins, newline, thirdparty, newline, project/local. So in this case, argparse, newline, yaml, newline, mcmahon. I am the king of nitpicks